lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -150,7 +146,7 @@ function Custom-GetFileHash ($Path, $Algorithm) {
 }
 
 function Get-SHA256Converter {
-  if (Is-FIPS) {
+  if ($(Is-FIPS) -ge 1) {
     New-Object -TypeName Security.Cryptography.SHA256Cng
```

Comment:
Hi @mwrock - I have verified on nano server and neither `New-Object -TypeName Security.Cryptography.SHA256Cng` nor `[System.Security.Cryptography.SHA256]::Create("System.Security.Cryptography.SHA256Cng")` working on that. Also If I just use `SHA256Cng` for all that is failing on `windows server 2008` because by default it comes with `powershell -v2.0` and `.NET Framework version 2.0`. As I already mention above `SHA265Cng` required `.NET Framework version 3.5 and above`. So let me know your thought on it.